tensorboard/logo/logo_circled.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 600 600"><defs><style>.cls-1{fill:url(#linear-gradient);}.cls-2{fill:#fff;}</style><linearGradient id="linear-gradient" x1="144.09" y1="300" x2="431.73" y2="300" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ff6f00"/><stop offset="1" stop-color="#ffa800"/></linearGradient></defs><title>TF_</title><g id="Layer_2" data-name="Layer 2"><circle class="cls-1" cx="300" cy="300" r="192.67"/><path class="cls-2" d="M299.5,141.55,188.89,205.32l16.45,9.49,94.16-54.26,120,69.28v43.52L436,263.86V220.33Z"/><polygon class="cls-2" points="314.44 448.42 360.37 421.9 360.37 291.91 314.44 318.43 314.44 448.42"/><polygon class="cls-2" points="390.15 404.52 436.08 378.01 436.08 291.88 390.15 318.4 390.15 404.52"/><polygon class="cls-2" points="235.03 387.96 188.97 317.5 163.2 325.81 163.2 379.58 284.73 449.53 284.73 340.42 235.03 387.96"/><polygon class="cls-2" points="196.55 296.07 238.26 359.87 284.73 315.41 284.73 292.59 163.2 222.64 163.2 306.83 196.55 296.07"/></g></svg>
```

Comment:
...`<title>TF_</title>`... I wonder if this is properly minified :\

Other extraneous strings are:
- `id="Layer_2"` 
- `data-name="Layer 2"`

Reply:
I noticed this, too, but I copied them directly from the design team, so
I’m inclined to check in a pristine copy first and run them through SVGO
afterward.